test/internals.js

```diff
@@ -171,6 +179,21 @@ const echoHandler = function (request, h) {
     return request.payload;
 };
 
+const sequentialHandler = async function (request, h) {
+
+    if (!sequentialHandler.callCount) {
+        sequentialHandler.callCount = 1;
+
+        await awaitDelay(500);
+
+        return sequentialHandler.callCount;
+    }
+
+    await awaitDelay(Math.floor(Math.random() * 100));
```

Comment:
I think in a perfect world the sequential handlers would count down: the first 40ms, the second 30ms, etc.

Reply:
Yeah, that's a good idea. I just wanted to make sure the first definitely was by far the longest, and added some response time jitter just to keep it interesting :P 

The main portion was the first -> second calls since `.callCount` would definitely 100% be incremented by then. I'll go ahead and do the delay decrement though. It's a good idea :+1:

Reply:
Oh!!  I actually missed that finer point.  I think ultimately we're good either way :)